src/screens/SignIn1Screen/SignIn1.tsx

```diff
@@ -36,30 +36,30 @@ const SignIn1Screen = ({ navigation }: any) => {
             Advancing Indigenous Peoples' Rights & Cultures Worldwide.
           </Text>
         </View>
-        {/* TODO: Potentially extract styles into the button component */}
+        {/* TODO @auth-flow: Potentially extract styles into the button component */}
```

Comment:
I like this @ auth-flow! good thinking : ) makes it much easier to search and resolve all the auth TODOs later